rotkehlchen/chain/evm/decoding/base.py

```diff
@@ -76,9 +76,14 @@ def decode_direction(
             self,
             from_address: ChecksumEvmAddress,
             to_address: Optional[ChecksumEvmAddress],
-    ) -> Optional[tuple[HistoryEventType, Optional[str], str, str]]:
+    ) -> Optional[tuple[HistoryEventType, Optional[str], ChecksumEvmAddress, str, str]]:
         """Depending on addresses, if they are tracked by the user or not, if they
-        are an exchange address etc. determine the type of event to classify the transfer as"""
+        are an exchange address etc. determine the type of event to classify the transfer as.
+
+        Returns event type, loation label, address, counterparty and verb.
```

Comment:
Forgot to fix this. I mentioned here: https://github.com/rotki/rotki/pull/5726/files#r1138539132

```suggestion
        Returns event type, location label, address, counterparty and verb.
```